specification/nhais-adaptor.yaml

```diff
@@ -185,291 +129,133 @@ paths:
-                              display: No patches submitted.
-                invalid-patch-operation:
-                  summary: Invalid patch operation(s)
-                  value:
-                    resourceType: OperationOutcome
-                    issue:
-                      - severity: error
-                        code: value
-                        details:
-                          coding:
-                            - system: 'https://fhir.nhs.uk/R4/CodeSystem/Spine-ErrorOrWarningCode'
-                              version: '1'
-                              code: invalidPatchOperation
-                              display: 'Invalid patch: attempted to modify Patient properties that cannot be changed by an Amendment'
-        '415':
-          description: Unsupported media type.
+                type: string
+              description: The unique identifier for the operation MUST be retained to match to the inbound reply
+        '400':
+          description: Bad Request - The request could not be processed and forwarded to NHAIS
           content:
             application/fhir+json:
               schema:
                 $ref: components/schemas/OperationOutcome.yaml
-              example:
-                resourceType: OperationOutcome
-                issue:
-                  - severity: error
-                    code: value
-                    details:
-                      coding:
-                        - system: 'https://fhir.nhs.uk/R4/CodeSystem/Spine-ErrorOrWarningCode'
-                          version: '1'
-                          code: unsupportedMediaType
-                          display: "Must be 'application/json-patch+json'."
-  '/Patient/$nhais.acceptance':
+  '/fhir/Patient/$nhais.acceptance':
     post:
       description: |
         ## Overview
         Use this endpoint to send an Acceptance to NHAIS.
 
-        ## Payload Description
-
-        The request payload is a Parameters list. The list must contain parameters with the following names and values:
-
-        * patient: a Patient `resource`, see below for details
-        * acceptanceType: simple parameter with `valueString` one of: birth, first, transferin, immigrant, exservices
-        * acceptanceCode: TODO
+        ## Supported Acceptance Types
 
-        ## Patient Parameter
+        * Type 1 - Birth
+        * Type 2 - First Acceptance
+        * Type 3 - Transfer In
+        * Type 4 - Immigrant
 
-        Refer to the examples to see how the fields are represented in the request body.
+        Type 5 - Ex-Services is no longer used. Such patients are now registered as Type 3 - Transfer In.
 
-        The following is paraphrased from:
+        ## Supported Fields
 
-        HA/GP LINKS - REGISTRATION, GP SYSTEMS SPECIFICATION
-
-        3.3.3.a  Algorithm to determine the Patient Acceptance Type
+        Refer to GP Links Specification Chapter 3 for the complete requirements for mantually-entered acceptances. The
```

Comment:
spelling - mantually

Reply:
didn't find it, I think I removed this section